conf/app.routes

```diff
@@ -371,4 +371,8 @@ POST       /change-agent-international-address                  controllers.Agen
 GET        /declaration                        controllers.DeclarationController.onPageLoad(mode: Mode = NormalMode)
 POST       /declaration                        controllers.DeclarationController.onSubmit(mode: Mode = NormalMode)
 
-GET       /confirmation                         controllers.ConfirmationController.onPageLoad()
\ No newline at end of file
+GET       /confirmation                         controllers.ConfirmationController.onPageLoad()
+
+GET        /your-answers-summary                          controllers.SummaryAnswerPageController.onPageLoad()
```

Comment:
Anita has asked if these could become `/your-draft-registration` and `/your-declared-registration`.

Reply:
Done